website/content/en/docs/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -11,7 +11,7 @@ Karpenter automatically provisions new nodes in response to unschedulable pods.
 
 This guide shows how to get started with Karpenter by creating a Kubernetes cluster and installing Karpenter.
```

Comment:
Also, can you make sure that the change you are proposing here is coming in the `preview` path rather than the docs path? The preview path is the one that we carry forward on new releases